android/app/src/main/java/app/organicmaps/search/SearchAdapter.java

```diff
@@ -266,4 +267,34 @@ void refreshData(@Nullable SearchResult[] results)
     mResults = results;
     notifyDataSetChanged();
   }
+
+  @Override
+    void bind(@NonNull SearchResult result, int order) {
+        super.bind(result, order);
+        setBackground();
+
+        formatOpeningHours(mResult);
+        UiUtils.setTextAndHideIfEmpty(mDescription, mResult.description.description);
+        UiUtils.setTextAndHideIfEmpty(mRegion, mResult.description.region);
+        UiUtils.setTextAndHideIfEmpty(mDistance, mResult.description.distance.toString(mFrame.getContext()));
```

Comment:
This code looks like a duplication of the existing functionality. How does it work? When is it called? How big is the overhead?